.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   pull_request:
-    types: [ready_for_review]
+    types: [review_requested]
```

Comment:
Noticed that this only runs when you update from `draft` but doesn't run when initially opening a PR for review